src/views/Player.vue

```diff
@@ -76,22 +94,37 @@ export default defineComponent({
           options: ["", "", ""],
           correct_answer: [0, 1],
           image: {
-            url: "https://plio-prod-assets.s3.ap-south-1.amazonaws.com/images/afbxudrmbl.png",
+            url:
+              "https://plio-prod-assets.s3.ap-south-1.amazonaws.com/images/afbxudrmbl.png",
             alt_text: "some image",
           },
           survey: true,
           max_char_limit: null,
         },
       ] as Question[],
       responses: [] as SubmittedResponse[], // holds the responses to each item submitted by the viewer
+      numCorrect: 0, // number of correctly answered questions
+      numWrong: 0, // number of wrongly answered questions
+      isScorecardShown: false, // to show the scorecard or not
     });
+    const isEqual = require("deep-eql");
     const isSplashShown = computed(() => state.currentQuestionIndex == -1);
     const isQuestionShown = computed(() => {
       return (
         state.currentQuestionIndex >= 0 &&
         state.currentQuestionIndex < state.questions.length
       );
     });
+    const showScorecard = computed(() => {
+      if (areAllQuestionsSurvey.value) return false;
+      if (state.currentQuestionIndex == state.questions.length) scorecard();
+      return state.currentQuestionIndex == state.questions.length;
+    });
```

Comment:
scorecard visibility should be controlled by `isScorecardShown` - why another variable?